src/actions/__test__/rover.test.js

```diff
@@ -1,14 +1,15 @@
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 import {
-  changeActiveRover,
+  connect,
   createRover,
+  disconnect,
   editRover,
   fetchRover,
   fetchRovers,
-  popCommand,
-  pushCommand,
```

Comment:
Not having a queue is nice. I think it will be cleaner to just enforce enough delay between things at a higher level.